README.md

```diff
@@ -264,6 +264,32 @@ colord("#00336680").toCmykString(); // "device-cmyk(100% 50% 0% 60% / 0.5)"
 
 </details>
 
+<details>
+  <summary><code>Harmonies</code></b> (<b>Harmony colors</b> plugin)</summary>
```

Comment:
Let's move it from Convert docs section to Manipulate.

Reply:
Got it!